console_test.go

```diff
@@ -69,11 +68,6 @@ func TestConsolePty(t *testing.T) {
 
 	iteration := 10
 
-	cmd := exec.Command("sh", "-c", fmt.Sprintf("for x in `seq 1 %d`; do echo -n test; done", iteration))
```

Comment:
not sure what the test is covering, rather than move the loop to go..

what about just changing this command to:
```
for x in 1 2 3 4 5 6 7 8 9 10; do printf test; done
```

Reply:
Great point! That works as well. I thought it might be important to keep the number of iterations more flexible. Overall, I don't see a clear advantage of one way over the other.